modules/touch/src/touchinteraction.cpp

```diff
@@ -249,6 +206,36 @@ namespace {
         "defaults to the surface of the current anchor."
     };
 
+    constexpr openspace::properties::Property::PropertyInfo
+        EnableDirectManipulationInfo =
+    {
+        "EnableDirectManipulation",
+        "Enable direct manipulation",
+        "Decides whether the direct manipulation mode should be enabled or not. "
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo
+        DirectManipulationThresholdInfo =
+    {
+        "DirectManipulationThreshold",
+        "Direct manipulation threshold",
+        "This threshold affects the distance from the interaction sphere at which the "
+        "direct manipulation interaction mode starts being active. The value is given "
+        "as a factor times the interaction sphere"
+    };
+
+    //constexpr openspace::properties::Property::PropertyInfo OrbitSpinningThreshold = {
```

Comment:
Do we still need the commented out code?

Reply:
It's related to a feature that I just commented out, but that we might want to bring back. So wasn't sure what to do with the properties

Reply:
My suggestion is to add a comment above that explains this and then keep it commented out.

Reply:
Ended up removing them, and all related code, completely. To limit the number of TODO comments in this class, and as we would not want to use this specific implementation anyways.